src/recs.js

```diff
@@ -72,10 +79,20 @@ export async function initTemplate(options) {
 	const templateDir = (answers && answers.directory) || templateDefaultDir;
 	const componentName = pascalCase(name);
 
+	const settings = {
+		type: `${TEMPLATE_TYPE_RECS}/${answers.type}`,
```

Comment:
This is failing when a nameArg is provided:
![image](https://user-images.githubusercontent.com/5060400/150381441-58ae762f-09dd-47c0-992b-b182e0b8fbb6.png)